app/src/main/java/ca/klapstein/baudit/models/RemoteModel.java

```diff
@@ -191,15 +265,14 @@ protected CareProvider doInBackground(String... search_parameters) {
             JestDroidClient client = createBaseClient();
             Log.d(TAG, "elastic search parameters: " + Arrays.toString(search_parameters));
             ArrayList<CareProvider> careProviderArrayList = new ArrayList<>();
-            String query =
-                    "{\n" +
-                            "    \"query\": {\n" +
-                            "        \"ids\" : {\n" +
-                            "            \"type\" : \"careprovider\",\n" +
-                            "            \"values\" : [\"" + search_parameters[0] + "\"]\n" +
-                            "         }\n" +
-                            "     }\n" +
-                            "}";
+            String query = "{\n" +
+                    "    \"query\": {\n" +
+                    "        \"ids\" : {\n" +
+                    "            \"type\" : \"careprovider\",\n" +
+                    "            \"values\" : [\"" + search_parameters[0] + "\"]\n" +
+                    "         }\n" +
+                    "     }\n" +
+                    "}";
```

Comment:
Maybe we could pass an extra type parameter to reduce code duplication?

Reply:
maybe add as a TODO: for later refactoring